source/docs/articles/sites/varnish/caching-advancedtopics.md

```diff
@@ -142,7 +142,7 @@ If you're using the Environment Access: Locked security setting on a site envir
 
 ## Pantheon's .vcl File
```

Comment:
Also change heading? Maybe  to "Pantheon's Varnish cookie handling" like in https://github.com/pantheon-systems/documentation/pull/301